README.md

```diff
@@ -125,6 +104,30 @@ Similar to logging, there is also another specific play for instrumenting with L
 This instrumentation is dependent on the application to generate a log file in its directory named `application.log.json`
 For example if a service was being deployed with the demo-deployer with an id of "node1" on an AWS EC2 instance, the log file should be in the path /home/ec2-user/node1/application.log.json
 
+### Alerts
+
+Alerts can be created in newrelic through the use of the Terraform provider.
```

Comment:
We should mention that the users needs to add config for managing an S3 bucket to the "resource" section of you config.

Reply:
Actually this is optional, folks don't have to create an S3 bucket with their deployment, and I suspect this may be the more frequent use case, particularly since buckets are limited on aws, I believe to 50 by default.
I could put a note that says you may create a bucket?

Reply:
I've put an additional note.